src/test/java/uk/gov/service/notify/NotificationClientTest.java

```diff
@@ -96,12 +96,17 @@ public void testPrepareUpload() throws NotificationClientException {
         assertEquals(expectedResult.getString("file"), response.getString("file"));
     }
 
-    @Test(expected = NotificationClientException.class)
-    public void testPrepareUploadThrowsExceptionWhenExceeds2MB() throws NotificationClientException {
+    public void testPrepareUploadThrowsExceptionWhenExceeds2MB(){
```

Comment:
doesn't this still need the `@test` decorator

Reply:
also we should put an `assertFalse(true);` or similar just after the line, so that if it doesn't throw an exception the test fails